presto-mysql/pom.xml

```diff
@@ -141,14 +141,8 @@
         </dependency>
 
         <dependency>
-            <groupId>io.airlift</groupId>
-            <artifactId>testing-mysql-server</artifactId>
-            <scope>test</scope>
-        </dependency>
-
-        <dependency>
```

Comment:
Are we missing a dependency on the `testcontainers` artifact?

Reply:
If we extend the class instead of just using it, it requires `testcontainers` artifact.
The current CI failure:
```
[WARNING] Used undeclared dependencies found:
[WARNING]    org.testcontainers:testcontainers:jar:1.12.4:test
```